### ISSUE TITLE
`@finos/perspective-webpack-plugin@0.10.0` has a stale peer dependency

### ISSUE THREAD

**Issue by zemeolotu** [finos/perspective#1503]
`@finos/perspective-webpack-plugin@0.10.0` has `@finos/perspective` as a peer dependency but is pegged to version  `^0.6.0` instead of `^0.10.0`

**Comment by texodus** [comment]
Yes, `lerna publish` annoyingly does not respect peer dependencies.
